### PR TITLE
[instabug-reactnative]add more declare

### PR DESCRIPTION
**Pull request by pengcao1**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by pengcao1** [comment]
anybody can review this?

**Comment by pengcao1** [comment]
@rbuckton  could you help me to review and merge this commit?

**Comment by pengcao1** [comment]
@PranavSenthilnathan 
could you help me to review and merge this commit?

File: types/instabug-reactnative/index.d.ts [hunk 1]
```diff
@@ -5,135 +5,235 @@
...
+  audio: string;
+  video: string;
+  image: string;
+  chatsHeaderTitle: string;
+  team: string;
+  messageNotification: string;
+  messagesNotificationAndOthers: string;
+
+  // iOS only
+  ollectingDataText: string;
```
Comment (PranavSenthilnathan) [inline types/instabug-reactnative/index.d.ts]: Is this a typo?

**Review by PranavSenthilnathan — CHANGES_REQUESTED** [review]

**Pull request closed without merging**